export/index.template.html

```diff
@@ -671,6 +689,111 @@ <h2>Message log</h2>
           return nobleEd25519.getPublicKey(privateKeyHex);
         }
 
+        /**
+         * Function to validate and sanitize the styles object using the accepted map of style keys and values (as regular expressions).
+         * Any invalid style throws an error. Returns an object of valid styles.
+         * @param {Object} styles
+         * @return {Object}
+         */
+         function validateStyles(styles, element) {
+          const validStyles = {};
+
+          const cssValidationRegex = {
+            padding: "^(\\d+(px|em|%|rem) ?){1,4}$",
```

Comment:
Might make sense to put these valid, modifiable css properties in a doc somewhere

Reply:
good idea, can add something to the UI customization page